Sources/DateExtensions.swift

```diff
@@ -32,10 +32,26 @@ extension Date {
             self = rfc850
             return
         }
-        if let asctime =  Date(fromString: httpDateString, format: "EEE MMM d HH':'mm':'ss yyyy") {
+        if let asctime = Date(fromString: httpDateString, format: "EEE MMM d HH':'mm':'ss yyyy") {
             self = asctime
             return
         }
+        if let iso8601DateOnly = Date(fromString: httpDateString, format: "yyyy-MM-dd") {
+            self = iso8601DateOnly
+            return
+        }
+        if let iso8601DateHrMinOnly = Date(fromString: httpDateString, format: "yyyy-MM-dd'T'HH:mmxxxxx") {
```

Comment:
B the way its 5 x.
It is to produce `+00:00` format for the timezone.
`ZZZZ` produces `+0000` without the `:`.

[Apple docs](https://developer.apple.com/reference/foundation/dateformatter) gave an example that `ZZZZZ` will give `+00:00` with the `:` but that does not seem to be the case.

I am getting this error with `ZZZZZ` for line 36:
`XCTAssertEqual(dateFromString.toString(format: self.format), self.dateString!)`
The error:
`test failure: -[DateTests testDateFromString()] failed: XCTAssertEqual failed: ("2015-11-14T16:05:11Z") is not equal to ("2015-11-14T16:05:11+00:00") - ` on line 36

I found 5x to work the way I needed.

I cant find anything on the 5 x from official Apple docs though :(